lte/cloud/go/services/lte/obsidian/models/swagger.v1.yml

```diff
@@ -1372,13 +1372,15 @@ paths:
         - EnodeBs
       parameters:
         - $ref: './orc8r-swagger-common.yml#/parameters/network_id'
+        - $ref: './orc8r-swagger-common.yml#/parameters/page_size'
+        - $ref: './orc8r-swagger-common.yml#/parameters/page_token'
       responses:
         '200':
-          description: All enodeBs registered in the network
+          description: All enodeBs registered in the network with pagination support
           schema:
             type: object
             additionalProperties:
-              $ref: '#/definitions/enodeb'
+              $ref: '#/definitions/paginated_enodebs'
```

Comment:
This isn't right. The response type is specifically paginated_enodebs. Right now you have ~ `anonymous_object{implicit: map[string]paginated_enodebs}`